package.json

```diff
@@ -22,5 +22,8 @@
   "devDependencies": {
     "husky": "^8.0.1",
     "lint-staged": "^13.0.3"
+  },
+  "dependencies": {
+    "dotenv": "^16.0.3"
```

Comment:
This should be installed inside the hardhat folder. 

This is a monorepo and we should see each folder a different projects / repo.

Reply:
fixes here https://github.com/scaffold-eth/se-2/pull/24/commits/0fc68d94d6087a33f827eec86a1f74aa80441e73